lib/srv/db/spanner_test.go

```diff
@@ -246,6 +246,10 @@ func TestAuditSpanner(t *testing.T) {
 		require.True(t, ok)
 		require.Equal(t, "googlesql", dbStart1.DatabaseName)
 
+		// the connection should start to shut down, but wait some time for that
+		// to happen before sending further RPCs so that the client will dial
+		// a new connection before sending another query RPC.
+		time.Sleep(3 * time.Second)
```

Comment:
any way to avoid the "long" sleep?

Reply:
I couldn't reproduce locally with a 10ms sleep, so I'll reduce this

Reply:
or maybe we wait util server has no action connections?